File: pyg-manager-web/src/main/webapp/js/service/specificationService.js
app.service("specificationService",function ($http) {
    this.findByPage=function (pageNum,pageSize) {
        return $http.get("../specification/findAllByPage?pageNum="+pageNum+"&pageSize="+pageSize);
    };
    this.findById=function (id) {
        return $http.get("../specification/findById?id="+id);
    };
    this.dele=function (ids) {
        return $http.get("../specification/delete?ids="+ids);
    };
    this.save=function (brand) {
        return $http.post("../specification/save",brand);
    };
    this.update=function (brand) {
        return $http.post("../specification/update",brand);
    }
});

File: pyg-manager/pyg-manager-interface/src/main/java/com.pyg.manager.service/SpecificationService.java
package com.pyg.manager.service;

import com.pyg.pojo.TbSpecification;
import com.pyg.utils.PageResult;
import com.pyg.vo.Specification;

import java.util.List;

/**
 * Created by on 2018/8/10.
 */
public interface SpecificationService {
    /**
     * 需求：查询所有品牌数据
     */
    public List<TbSpecification> findAll();

    PageResult findAllByPage(int pageNum, int pageSize);

    public void save(Specification specification);

    public Specification findById(long id);

    public void update(Specification specification);

    public void deleteById(long id);
}
